Sming/Libraries/ModbusMaster/ModbusMaster.patch

```diff
@@ -1,8 +1,27 @@
 diff --git a/src/ModbusMaster.cpp b/src/ModbusMaster.cpp
-index cdcc534..f6f93d4 100644
+index cdcc534..ecb03a5 100644
 --- a/src/ModbusMaster.cpp
 +++ b/src/ModbusMaster.cpp
-@@ -860,7 +860,7 @@ uint8_t ModbusMaster::ModbusMasterTransaction(uint8_t u8MBFunction)
+@@ -756,15 +756,13 @@ uint8_t ModbusMaster::ModbusMasterTransaction(uint8_t u8MBFunction)
+   // flush receive buffer before transmitting request
+   while (_serial->read() != -1);
+ 
+-  // transmit request
+   if (_preTransmission)
+   {
+     _preTransmission();
+   }
+-  for (i = 0; i < u16ModbusADUSize; i++)
+-  {
+-    _serial->write(u8ModbusADU[i]);
+-  }
++  constexpr uint8_t NUL{0};
++  _serial->write(u8ModbusADU, u16ModbusADUSize);
++  _serial->write(NUL);
```

Comment:
Remove `constexpr uint8_t NUL{0};`. And then replace `_serial->write(NUL);` with `_serial->write((uint8_t)0);